react/package.json

```diff
@@ -31,7 +31,7 @@
     ]
   },
   "engines": {
-    "npm": ">=8.0.0 <9.0.0",
```

Comment:
Will this break running with hot reload using `npm start`?
if there is a different command now we should update the docs
https://github.com/sentry-demos/empower?tab=readme-ov-file#option-2-run-react-app-w-hot-reload

Reply:
Resolved in https://github.com/sentry-demos/empower/pull/413/commits/922faa11df3c75acdbf1da96a5b082571c99f54b